src/utils/regexp.js

```diff
@@ -0,0 +1,4 @@
+export const checkToOnlyNumber = (value) => /^-?\d*$/.test(value);
+
+export const moneyFormat = (value) =>
+  value.replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
```

Comment:
Need to perfomance test this regex with the solution that we've created:

**Solution:**
```javascript
const digits = Array.from(String(number)).reverse().join(''); /// O(3*n ~ n)
const groups = digits.match(/\d{1, 3}/g).reverse().join(','); /// O(3*n ~ n) > O(n)*
```

Also, it could be optimized as well, because i wrote it fast and wierd.